homework/lesson6/Lesson6homework1Andre.py

```diff
@@ -1,9 +1,12 @@
-Words = ["hello", "world", "of", "tanks", "video", "games", "phantom", "forces", "stupid", "hackers"]
-user = input("Please input a word (no capitals): ")
-if (user in Words) is False:
-    Words.append(user)
-    print("Added new word to array")
-    print(Words)
-else:
-    print("The word you just entered is already in the system")
+def array_func():
+    Words = ["hello", "world", "of", "tanks", "video", "games", "phantom", "forces", "stupid", "hackers"]
+    user = input("Please input a word (no capitals): ")
+    if (user in Words) is False:
+        Words.append(user)
+        print("Added new word to array")
+        print(Words)
+    else:
+        print("The word you just entered is already in the system")
+
```

Comment:
2 empty lines are recommended because some editors recognize the main program only after 2empty lines